example/docker/docker-compose.yml

```diff
@@ -30,6 +30,7 @@ services:
       - HONO_SERVER_BIND_ADDRESS=0.0.0.0
       - HONO_SERVER_KEY_STORE_PATH=/etc/hono/certs/honoKeyStore.p12
       - HONO_SERVER_KEY_STORE_PASSWORD=honokeys
+      - HONO_SERVER_ALLOW_INSECURE=false
```

Comment:
Correct - I wanted to make the flag (and only the flag) visible in the central compose file.
I am ambivalent on this - would you tend to remove it (as not being necessary) again?